__init__.py

```diff
@@ -64,10 +64,3 @@ def stop(self):
         If not relevant to your skill, feel free to remove.
         """
         pass
-
-
-def create_skill():
-    """Optional for OVOSSkill, required for MycroftSkill.
-    Left for backwards compatibility reasons but it can be safely removed.
-    """
-    return HelloWorldSkill()
```

Comment:
@JarbasAl to my knowledge this will prevent using this skill as a git clone in `skills`

Reply:
it shouldn't, the file loader also detects the class object directly

it wont work in classic mycroft-core, but we dont care about that....

Reply:
They have their own Hello world skill. 🙂